ccdaservice/README.md

```diff
@@ -38,7 +38,7 @@ Navigate to: openemr/ccdaservice and run the following from an elevated PowerShe
 - `npm install --global --omit=dev windows-build-tools` (Recommended) Though compiling libxmljs is no longer required, if for some reason your environment doesn't have download  compiled libxmljs2 binary available then npm will try to compile.
 - `npm install --omit=dev`
 ### Use
-* CCDA service must be enabled in OpenEMR's menu Globals->Connectors.
+* CCDA service must be enabled in OpenEMR's menu Admin->Config->Connectors->Enable C-CDA Service.
 * This service will automatically start on demand when required by OpenEMR.
```

Comment:
Can you change the upgrade instruction for update/upgrading using npm install followed by npm ci to stress the importance of following that workflow. Also add that it is necessary to kill all node processes or at least the ones using port 6660,6661 and 6662.
It may be worthwhile to add scripts to auto kill node before install or provide a batch/bat.
I'd prefer changing back to 127.0.0.1 as I know that to be working since start of the service where localhost could dig up some unexpected issues. Although docker and proxies probably need research for what is recommended.

You can change that the service has be tested on 20.1 and v18 for new libxmljs2 version. Thanks